.github/workflows/conformance-datapath.yaml

```diff
@@ -186,21 +186,14 @@ jobs:
             fi
 
       - name: Run tests
-        uses: cilium/little-vm-helper@76cb7b131c9fa60f697af29106b529c0a423a17e
+        uses: cilium/little-vm-helper@2a076423707ff588c2b0ed676d835e779b4c4af4
         with:
           test-name: datapath-conformance
-          image: kind
-          lvh-version: 'v0.0.1'
           image-version: ${{ matrix.kernel }}
           host-mount: ./
           cmd: |
-            set -eux
-
-            echo 'nameserver 1.1.1.1' > /etc/resolv.conf
```

Comment:
The commit description doesn't explain why this isn't needed anymore.

Reply:
Good catch, added to the commit msg.